static/js/components/tiles/sv_ranking_units.tsx

```diff
@@ -216,6 +215,33 @@ export function getRankingUnitTitle(
   return formatString(title, rs);
 }
 
+/**
+ * Gets the top points and bottom points to display in a ranking unit
+ * @param rankingMetadata the RankingTileSpec to get the points for
+ * @param isHighest whether or not this ranking unit is showing highest
```

Comment:
nit: "...is showing highest" reads like it cuts too early. "...showing highest points"?

Reply:
done